### PR TITLE
Ins 1279 jet tree updater improvement

### PR DESCRIPTION
**Pull request by ruz**

### CODE REVIEW

File: ledger/artifactmanager/jettreeupdater.go [hunk 1]
```diff
@@ -128,23 +136,40 @@ func (jtu *jetTreeUpdater) fetchJet(
...
-type result struct {
-	jet *core.RecordID
-	err error
+func (jtu *jetTreeUpdater) releaseJet(ctx context.Context, jetID core.RecordID, pulse core.PulseNumber) {
+	jtu.seqMutex.Lock()
+	defer jtu.seqMutex.Unlock()
+
+	depth, _ := jet.Jet(jetID)
+	for {
+		key := fmt.Sprintf("%d:%s", pulse, jetID.String())
```
Comment (nordicdyno) [inline ledger/artifactmanager/jettreeupdater.go]: not nice, maybe struct be better?

File: ledger/artifactmanager/jettreeupdater.go [hunk 2]
```diff
@@ -109,7 +110,10 @@ func (jtu *jetTreeUpdater) fetchJet(
 		jtu.seqMutex.Unlock()
 	}()
 
-	resJet, err := jtu.fetchActualJetFromOtherNodes(ctx, target, pulse)
+	ch := jtu.fetchActualJetFromOtherNodes(ctx, target, pulse)
```
Comment (nordicdyno) [inline ledger/artifactmanager/jettreeupdater.go]: it's possible to avoid returning channel there

Comment (ruz) [reply ledger/artifactmanager/jettreeupdater.go]: done

File: ledger/artifactmanager/jettreeupdater.go (hunk 1 above)
Comment (ruz) [reply ledger/artifactmanager/jettreeupdater.go]: done

**Pull request merged**